packages/beyondinsight_password_safe/data_stream/session/manifest.yml

```
@@ -0,0 +1,40 @@
+title: BeyondInsight and Password Safe Session Events
+type: logs
+streams:
+  - input: cel
+    title: BeyondInsight and Password Safe Session Events
+    description: Request settings for BeyondInsight and Password Safe sessions.
+    enabled: true
+    template_path: cel.yml.hbs
+    vars:
+      - name: interval
+        type: text
+        title: Interval
+        description: Data for all sessions will be refetched at this interval.
+        show_user: false
+        required: true
+        default: "60s"
```

```suggestion
        default: 5m
```
Please make this the default.